src/test/java/io/quarkus/bot/it/IssueOpenedTest.java

```diff
@@ -32,5 +34,4 @@ void triage() throws IOException {
                     verifyNoMoreInteractions(mocks.ghObjects());
                 });
     }
-
```

Comment:
All these changes shouldn't be there. Have you reviewed your PR before creating it? I'll clean it up once you fixed the constant issue.